frontend/src/components/HeaderLogo.tsx

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Why do we have this empty file?

Reply:
I am having issues with compling
`./src/components/ConditionalNavbar.tsx:4:20
Type error: File '/Users/albin/dev/raise/le-commit/frontend/src/components/HeaderLogo.tsx' is not a module.

  2 |
  3 | import { usePathname } from 'next/navigation';
> 4 | import Navbar from './HeaderLogo';
    |                    ^
  5 |
  6 | export default function ConditionalNavbar() {
  7 |   const pathname = usePathname();
Next.js build worker exited with code: 1 and signal: null`